index.html

```diff
@@ -27,7 +27,115 @@ <h1>Charlie Giorgania</h1>
         </p>
         <button class="btn">Start Collaboration</button>
         <img class="combo" alt="" />
+        <h2>Projects</h2>
       </div>
     </section>
+
+    <section class="work-section">
+      <div class="grid-container">
+        <img class="mac" alt="" />
+        <img class="mac2" alt="" />
+        <img class="mac3" alt="" />
+        <img class="mac4" alt="" />
+        <img class="mac5" alt="" />
+        <img class="mac6" alt="" />
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="abtme">
+          <h1>About</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ea eiusmod
+          </p>
+          <button>Get my cv</button>
+        </div>
```

Comment:
- [ ]  Hey, kindly note that the about section is another stand-alone section from the design and you should not wrap it inside the work section. please, kindly move the `<div class="abtme"` out of the works section to improve your page layout.
- [ ] The container for the about section is a `<section>` to enforce a better use of the semantic tag. let's improve on this